spec/lib/services/obm-api-service-spec.js

```diff
@@ -6,7 +6,6 @@
 describe("Http.Services.Api.Obms", function () {
     var obmService;
     var Promise;
-    var chai = require('chai');
     chai.use(require('chai-string'));
```

Comment:
Just open a PR for on-core to move `chai-string` to on-core: https://github.com/RackHD/on-core/pull/94